scripts/getsamplesummary.py

```diff
@@ -39,57 +24,113 @@ def getsampleinfofromname(pars, sample):
-                   "_" + nameparts[4][-1:] + ".fastq.gz")
-        print fastqfile
-        print newname
+        nameparts = fastqfile.split("/")[-1].split("_")
+        rundir = fastqfile.split("/")[6]
+        date = rundir.split("_")[0]
+        fc = rundir[-9:]
+        newname = "{lane}_{date}_{fc}_{sample_name}_{index}_{readdirection}.fastq.gz".format(
+          lane=nameparts[3][-1:],
+          date=date,
+          fc=fc,
+          sample_name=sample_name,
+          index=nameparts[2],
+          readdirection=nameparts[4][-1:]
+        )
+        print(fastqfile)
+        print(os.path.join(outputdir, 'exomes', sample_name, 'fastq', newname))
         try:
-          os.symlink(fastqfile, outputdir + samplename + "/fastq/" + newname)
+          os.symlink(fastqfile, os.path.join(outputdir, 'exomes', sample_name, 'fastq', newname))
         except:
-          print "Can't create symlink for " + samplename
+          print("Can't create symlink for {}".format(sample_name))
+        try:
+          if cust_name != None and family_id != None:
+            mip_outdir = os.path.join(outputdir, cust_name, family_id, 'exomes')
+            link_source = os.path.join(outputdir, 'exomes', sample_name)
+            print('mkdir {}'.format(mip_outdir))
 
-smpls = getsamplesfromflowcell(params, fc)
+            os.makedirs(mip_outdir)
+            os.symlink(link_source, os.path.join(mip_outdir, sample_name))
+        except:
+          print("Can't create symlink for {} in MIP_ANALYSIS/cust".format(sample_name))
+
+def main(argv):
 
-for sample in smpls.iterkeys():
-  with lims.limsconnect(params['apiuser'], params['apipass'], params['baseuri']) as lmc:
-    pure_sample = sample.rstrip('BF') # remove the reprep (B) and reception control fail (F) letters from the samplename
-    analysistype = lmc.getattribute('samples', pure_sample, "Sequencing Analysis")
-    readcounts = .75 * float(analysistype[-3:])    # Accepted readcount is 75% of ordered million reads
-  dbinfo = getsampleinfofromname(params, pure_sample)
-  rc = 0         # counter for total readcount of sample
-  fclanes = {}   # dict to keep flowcell names and lanes for a sample
-  cnt = 0        # counter used in the dict to keep folwcell/lane count
-  for info in dbinfo:
-    if (info['q30'] > 80):     # Use readcount from lane only if it satisfies QC [=80%]
-      cnt += 1
-      rc += info['M_reads']
-      fclanes[cnt] = info['fc'] + "_" + str(info['q30']) + "_" + str(info['lane'])
-  if readcounts:
-    if (rc > readcounts):        # If enough reads are obtained do
-      print pure_sample + " Passed " + str(rc) + " M reads\nUsing reads from " + str(fclanes)
-      makelinks(pure_sample, fclanes)
-    else:                        # Otherwise just present the data
-      print pure_sample + " Fail " + str(rc) + " M reads\nThese flowcells summarixed " + str(fclanes)
+  outputdir = '/mnt/hds/proj/bioinfo/MIP_ANALYSIS/'
```

Comment:
Yep, you are spot on. I've started work on the central config class in Clinical-Genomics/cghq#4. I think the best plan is make it possible to just pass this value in as an argument to a function.